src/xtgeo/interfaces/rms/rmsapi_utils.py

```diff
@@ -0,0 +1,450 @@
+"""Module for various operation in the RMSAPI (former rmsapi) python interface."""
+
+import enum
+import pathlib
+import warnings
+from typing import Any, Literal
+
+from packaging.version import parse as versionparse
+
+from xtgeo.common.log import null_logger
+
+from ._rmsapi_package import RmsProjectType, rmsapi
+
+logger = null_logger(__name__)
+
+__all__ = [
+    "RmsApiUtils",
+    "RoxUtils",  # For backward compatibility
+    "_DomainType",
+    "_DomainTypeClipBoardGeneral2D",
+    "_StorageTypeRegularSurface",
+]
```

Comment:
Similar here. The main use case for `__all__` is `from .module import *` which is generally not recommended anyway